blueprint/core/blueprint_ShadowView.h

```diff
@@ -1,163 +1,170 @@
+                return true;
+            }
+
+            setter(node, args..., YGUndefined);
+            return true;
+        };
     }
 
     template <typename Setter, typename SetterPercent, typename SetterAuto, typename ...Args>
-    const auto getYogaNodeDimensionAutoSetter(Setter setter, SetterPercent setterPercent, SetterAuto setterAuto, Args... args) {
-      return [=, nonAutoSetter = getYogaNodeDimensionSetter(setter, setterPercent, args...)](const juce::var& value, YGNodeRef node) {
-        if (value.isString() && value.toString() == "auto") {
-          setterAuto(node, args...);
-          return true;
-        }
-        return nonAutoSetter(value, node);
-      };
+    const auto getYogaNodeDimensionAutoSetter (Setter setter, SetterPercent setterPercent, SetterAuto setterAuto, Args... args)
+    {
+        return [=, nonAutoSetter = getYogaNodeDimensionSetter(setter, setterPercent, args...)] (const juce::var& value, YGNodeRef node)
+        {
+            if (value.isString() && value.toString() == "auto")
+            {
+                setterAuto (node, args...);
+                return true;
+            }
+
+            return nonAutoSetter (value, node);
+        };
     }
 
     template <typename Setter, typename EnumMap>
-    const auto getYogaNodeEnumSetter(Setter setter, EnumMap &map) {
-      return [=](const juce::var& value, YGNodeRef node) {                       \
-        const auto val = map.find(value);
-        if(val == map.end()) {
-          // TODO catch further up to add the key at which we tried
-          // to set this enum property to the message and rethrow
-          throw std::invalid_argument("Invalid property: " + value.toString().toStdString());
-        }
-        setter(node, val->second);
-        return true;
-      };
+    const auto getYogaNodeEnumSetter (Setter setter, EnumMap &map)
+    {
+        return [=] (const juce::var& value, YGNodeRef node)
+        {
+            const auto val = map.find (value);
+            if (val == map.end())
+            {
+                // TODO catch further up to add the key at which we tried
+                // to set this enum property to the message and rethrow
+                throw std::invalid_argument ("Invalid property: " + value.toString().toStdString());
+            }
+
+            setter (node, val->second);
+            return true;
+        };
     }
 
     //==============================================================================
     /** The ShadowView class decouples layout constraints from the actual View instances
         so that our View tree and ShadowView tree might differ (i.e. in the case of raw
         text nodes), and so that our View may remain more simple.
-     */
+    */
     class ShadowView
     {
     public:
         //==============================================================================
-        ShadowView(View* _view) : view(_view)
+        /** */
+        ShadowView(View* v) :
```

Comment:
This is just my POV but seeing that we have a .cpp file going, it might be better to move `ShadowView`'s method bodies into it...

Reply:
+ 1. There's a view headers that might benefit from being split now.